### PR TITLE
Add APT package build and release

### PR DESCRIPTION
**Pull request by volllly**

### CODE REVIEW

**Review by forsthug — DISMISSED** [review]

File: src/fiskaltrust.Launcher/fiskaltrust.Launcher.csproj [hunk 1]
```diff
@@ -6,6 +6,13 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+
+    <DefineConstants></DefineConstants>
+    <DefineConstants></DefineConstants>
```
Comment (TSchmiedlechner) [inline src/fiskaltrust.Launcher/fiskaltrust.Launcher.csproj:11]: I guess once would be enough :D

**Review by TSchmiedlechner — APPROVED** [review]

**Comment by TSchmiedlechner** [comment]
Great work! 🌮

File: azure-pipelines/templates/deb/release.yml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+stages:
+- stage: Release_Deb
+  displayName: Release deb Packages
+  dependsOn: []
+
+  jobs:
+  - deployment: Release_Deb
+
+    # condition: startsWith(variables['Build.SourceBranch'], 'refs/tags/v')
```
[the remark this answers is not in the document]
Comment (volllly) [reply azure-pipelines/templates/deb/release.yml:27]: I'm not sure if we want that 🤔 but probably we do ^^

File: README.md [hunk 3]
```diff
@@ -41,6 +41,22 @@ To stop the Launcher press <kbd>Ctrl</kbd> + <kbd>C</kbd>.
 > fiskaltrust.Launcher.exe --help
 > ```
 
+### Installation
+
+On debian based linux systems the Launcher can also be installed via `apt-get`.
+
+```bash
+curl -L http://downloads.fiskaltrust.cloud/apt-repo/KEY.gpg | gpg --dearmor | sudo tee /usr/share/keyrings/fiskaltrust-archive-keyring.gpg > /dev/null
```
Comment (TSchmiedlechner) [inline README.md:49]: This silently gets stuck on my Ubuntu when I don't add `sudo` 😅

**Review by TSchmiedlechner — APPROVED** [review]

**Pull request merged**